src/mobu/handlers/external.py

```diff
@@ -104,6 +111,20 @@ async def get_flock(
     return context.manager.get_flock(flock).dump()
 
 
+@external_router.put(
+    "/flocks/{flock}",
+    responses={404: {"description": "Flock not found", "model": ErrorModel}},
+    status_code=202,
+    summary="Signal a flock to refresh",
+)
+async def refresh_flock(
+    flock: str,
+    context: Annotated[RequestContext, Depends(context_dependency)],
+) -> None:
+    context.manager.refresh_flock(flock)
+    context.logger.info("Signaling flock to refresh", flock=flock)
```

Comment:
I don't love that `PUT` means refresh, but something like `GET /flocks/{flock}/refresh` doesn't seem to fit with the style.  I'm definitely open to other suggestions!